StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using TechTalk.SpecFlow;
+
+namespace TestSpecs {
+    [Binding]
+    class TestStpes
+    {
+        [Given(@"I am on the (.*) website")]
+        public void GivenIAmOnTheNLWebsite(string countryCode)
+        {
+            setCountryCode(countryCode);
+            Driver.Navigate().GoToUrl(baseURLCK);
```

Comment:
I don't like to use `Driver` directly,  it's better to use it within an `TestContext`